composer.json

```diff
@@ -13,7 +13,9 @@
         "sparkphp/command": "self.version"
     },
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "equip/assist": "^0.1.2",
+        "equip/data": "^2.3"
```

Comment:
With the changes in this PR it does, yes. I [noted separately](https://github.com/equip/command/pull/6#issuecomment-205863177) that I'm not convinced of the value of this PR over the functionality it would deprecate.